policy/modules/system/libraries.if

```diff
@@ -233,6 +233,31 @@ interface(`libs_dontaudit_write_lib_dirs',`
 	dontaudit $1 lib_t:dir write;
 ')
 
+########################################
+## <summary>
+##	Do not audit attempts to manage to library directories.
+## </summary>
+## <desc>
+##	<p>
+##	Do not audit attempts to manage to library directories.
+##	Typically this is used to quiet attempts to recompile
+##	python byte code.
+##	</p>
+## </desc>
+## <param name="domain">
+##	<summary>
+##	Domain to not audit.
+##	</summary>
+## </param>
+#
+interface(`libs_dontaudit_manage_lib_dirs',`
+	gen_require(`
+		type lib_t;
+	')
+
+	dontaudit $1 lib_t:dir manage_dir_perms;
```

Comment:
I worry that this (and the files version) will cause pain by silencing more innocuous permissions like getattr and read.  I guess most domains have those anyways, so maybe it's not a big deal, but I wonder if silencing the actually disallowed permissions (write etc) would be more appropriate?

Reply:
Since `domain` has `list_dir_perms`, I do not have this concern.